honestcash-v2/src/app/modules/editor/components/paid-section-selection/paid-section-selection.component.html

```diff
@@ -11,19 +11,21 @@
       <small><b>Cost in BCH:</b></small>
     </label>
     <input
-      class="form-control"
       name="elPaidSectionCostInBch"
       #elPaidSectionCostInBch="ngModel"
-      [type]="'number'"
+      [(ngModel)]="story.paidSectionCost"
+      (ngModelChange)="onChangePaidSectionCost()"
       [step]="0.001"
       [min]="0.001"
+      [max]="1"
       [value]="0.001"
```

Comment:
Can we make the step size bigger so the selection is somehow easier?
E.g
[step]="0.005"
[min]="0.005"
[max]="0.100"